tests/integration/IntegrationTest.php

```diff
@@ -1300,4 +1300,15 @@ public function testUnreadCountsBatch()
         $this->assertNotEmpty($resp["counts_by_user"][$this->user1["id"]]["total_unread_threads_count"]);
         $this->assertEquals(1, $resp["counts_by_user"][$this->user1["id"]]["total_unread_threads_count"]);
     }
+
+    public function testChannelPin()
+    {
+        $this->channel->addMembers([$this->user1["id"]]);
+        $this->channel->addMembers([$this->user2["id"]]);
+
+        // Pin the channel
+        $response = $this->channel->pin($this->user1["id"]);
+        print_r($response);
+        $this->assertNotNull($response["member"]["channelMember"]["pinned_at"]);
```

Comment:
test is failing, seems like the key should be `channel_member` and not `member`?

Reply:
yeah still wip